lms/templates/problem.html

```diff
@@ -36,7 +36,9 @@ <h3 class="hd hd-2 problem-header" tabindex="-1">
     % endif
     % if answer_available:
     <span class="problem-action-button-wrapper">
-        <button class="show problem-action-btn btn-default btn-small"><span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
+        <button class="show problem-action-btn btn-default btn-small">
```

Comment:
I suggested to @alisan617 that we put an aria-describedby attribute on this button that references the number of attempts left. That means we will need to generate a unique ID on that element.  During the regular workflow, it would be important that the learner be informed of how many attempts they have left when they focus on the submit button.